src/ntp/Makefile

```diff
@@ -38,8 +38,20 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# and go into learning mode.
 	sed -i 's/\/usr\/sbin\/ntpd {/\/usr\/sbin\/ntpd flags=(attach_disconnected complain) {/' debian/apparmor-profile
 
+ifeq ($(CROSS_BUILD_ENVIRON), y)
```

Comment:
Hi, Did you encounter this error while compiling for armhf?

```
make[6]: Entering directory '/sonic/src/ntp/ntp-4.2.8p12/include/isc'
make[6]: Nothing to be done for 'all'.
make[6]: Leaving directory '/sonic/src/ntp/ntp-4.2.8p12/include/isc'
make[6]: Entering directory '/sonic/src/ntp/ntp-4.2.8p12/include'
make[6]: *** No rule to make target 'adjtime.h', needed by 'all-am'.  Stop.
make[6]: Leaving directory '/sonic/src/ntp/ntp-4.2.8p12/include'
make[5]: *** [Makefile:616: all-recursive] Error 1
make[5]: Leaving directory '/sonic/src/ntp/ntp-4.2.8p12/include'
make[4]: *** [Makefile:667: all-recursive] Error 1
make[4]: Leaving directory '/sonic/src/ntp/ntp-4.2.8p12'
make[3]: *** [Makefile:599: all] Error 2
make[3]: Leaving directory '/sonic/src/ntp/ntp-4.2.8p12'
dh_auto_build: make -j64 returned exit code 2
```

Reply:
No. Does it happen in arm64 build ?

Reply:
Yes

Reply:
I'll check it.

Reply:
I just built successfully ntp debian package for arm64. 
I noticed that the created Makefile.in as well as Makefile in src/ntp/ntp-4.2.8p12/include directory does not contain adjtime.h in the noinst_HEADERS variable however Makefile.am does contain it in the noinst_HEADERS variable. So it looks like automake removed adjtime.h file from the noinst_HEADERS variable when building Makefile.in .
Hope this helps.

Reply:
I've dug through a bit and then found a patch which removed adjtime.h line from the Makefile.am. 

Anyway, i've ran a make clean followed by this build and it worked :) Thanks

Reply:
Great ! Is there anything else that prevents this PR merging ?

Reply:
I've only tested the changes you've made for arm64 (not armhf). For now, i could successfully build the buster slave-docker and most of the non-platform specific debs. I've gave comments on the debs which are failing. I've not yet tested for the files/ & whl/ and docker packages yet.

But as for the overall scope of this PR, i'm not the official reviewer and couldn't sign-off  

As a general advice, If the CI for this PR could include a build for marvell-armhf with cross compilation flags enabled, the reviewers will have confidence in merging the PR. @lguohan, is it possible?